Spectacle/Sources/SpectacleWindowPositionCalculator.m

```diff
@@ -203,6 +204,54 @@ - (BOOL)isWindowRect:(CGRect)windowRect tooSmallRelativeToVisibleFrameOfScreen:(
 
 #pragma mark -
 
+- (SpectacleCalculationResult *)calculateCornerRect:(CGRect)windowRect
+                                            visibleFrameOfScreen:(CGRect)visibleFrameOfScreen
+                                                      withAction:(SpectacleWindowAction)action
```

Comment:
Can we align these arguments properly? Lines 208 and 209 should have their colons aligned with the colon on line 207.